src/realtime_metrics/main.py

```diff
@@ -84,21 +95,26 @@ def mse_accuracy(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]]):
         trip_update = update[0]
         stop_time_update = update[1]
         actual_delay = get_actual_delay(trip_update=trip_update, stop_time_update=stop_time_update)
+        if actual_delay is None:
+            continue # skip updates with unknown actual delay
         predicted_delay = stop_time_update.arrival_delay
         prediction_error = actual_delay - predicted_delay
         samples.append(prediction_error)
 
     # compute MSE
-    sum = 0
+    if len(samples) <= 0:
```

Comment:
length smaller then 0 is not possible, so I was confused when initially reading this. We might it replace it like this:
```suggestion
    if len(samples) == 0:
```